cypress/integration/sales_order_spec.js

```diff
@@ -1,21 +1,50 @@
+import config from '../config';
+
 describe('New sales order test', function() {
+  const windowId = 143;
+  let caption = '';
+  let nodeId = null;
+  let menuOption = '';
+
   before(function() {
-    // login before each test
     cy.loginByForm();
-    cy.visit('/window/143');
+
+    cy.request('GET', config.API_URL+'/menu/elementPath?type=window&elementId='+windowId+'&inclusive=true')
+      .then((response) => {
+        expect(response.body).to.have.property('captionBreadcrumb');
+        expect(response.body).to.have.property('nodeId');
+        caption = response.body.captionBreadcrumb;
+        nodeId = response.body.nodeId;
+
+        cy.request('GET', config.API_URL+'/menu/node/'+nodeId+'/breadcrumbMenu')
+          .then((response) => {
+            const resp = response.body;
+            expect(resp.length).to.be.gt(0);
+
+            for (let i=0; i<resp.length; i+= 1) {
+              if (resp[i].nodeId === '1000040-new') {
```

Comment:
what is the `1000040` about? is it a constant value? i'm not sure, it might be incremenented on each new order..

Reply:
I think it's not but I just changed it to look for `-new`.